Manager/src/main/java/io/raspberrywallet/manager/linux/IndicatorLED.java

```diff
@@ -0,0 +1,90 @@
+package io.raspberrywallet.manager.linux;
+
+import com.pi4j.io.gpio.*;
+import com.pi4j.wiringpi.Gpio;
+import lombok.Getter;
+import lombok.Setter;
+
+public class IndicatorLED {
+
+    private final Pin ledPin = RaspiPin.GPIO_05;
+    private final GpioController gpioController = GpioFactory.getInstance();
+    private GpioPinDigitalOutput ledOutput;
+
+    public IndicatorLED() {
+        ledOutput = gpioController.provisionDigitalOutputPin(ledPin);
+    }
+
+    private Thread blinkThread;
+    private BlinkRunnable blinkRunnable;
+
+    private class BlinkRunnable implements Runnable {
+
+        @Setter private int offMs = 50;
+        @Setter private int onMs = 50;
+        @Setter private boolean work = true;
```

Comment:
volatile ?

Reply:
No need, thread will be run only in one instance, so the LED should. In case future, there would be a setter for GPIO, so you can mount more leds.